cmd/manager/main.go

```diff
@@ -215,12 +226,15 @@ func main() {
 			cmd.FatalError(setupLogger, err, "unable to create controller", "name", controllers.JobGCReconcilerName)
 		}
 
-		preflightStatusUpdaterAPI := preflight.NewStatusUpdater(client)
-		preflightAPI := preflight.NewPreflightAPI(client, buildAPI, signAPI, registryAPI, preflightStatusUpdaterAPI, kernelAPI)
+		//[TODO] - update the preflight flow with the MIC/MBSC implementation and then uncomment the preflight conroller
+		/*
+			preflightStatusUpdaterAPI := preflight.NewStatusUpdater(client)
+			preflightAPI := preflight.NewPreflightAPI(client, buildAPI, signAPI, registryAPI, preflightStatusUpdaterAPI, kernelAPI)
 
-		if err = controllers.NewPreflightValidationReconciler(client, filterAPI, metricsAPI, preflightStatusUpdaterAPI, preflightAPI).SetupWithManager(mgr); err != nil {
-			cmd.FatalError(setupLogger, err, "unable to create controller", "name", controllers.PreflightValidationReconcilerName)
-		}
+			if err = controllers.NewPreflightValidationReconciler(client, filterAPI, metricsAPI, preflightStatusUpdaterAPI, preflightAPI).SetupWithManager(mgr); err != nil {
+				cmd.FatalError(setupLogger, err, "unable to create controller", "name", controllers.PreflightValidationReconcilerName)
+			}
+		*/
```

Comment:
Can we do it in the same PR? If you insist on keeping small changes then I would do a multi-commit PR but I think it makes sense to put all of the following inside the same PR and it will be easier for me to review as well:
* Create the new controllers
* Remove the old controllers
* Use the new buildsign pkg everywhere needed
* Remove the old build/sign packages

Reply:
it would be better to do it in a separate PR: preflight is not a major priority, and i am not yet sure how many changes i would need to make in order to make it work.